docs/building-your-quest/actions/network-http-request.md

```diff
@@ -40,20 +40,20 @@ These can be accessed using:
 ```yaml
 do:
 - actionId: network_http_request
-  name: call_heroku_backend
+  name: call_k8s_backend
   params:
-    url: "https://${user.backendHerokuAppName}.herokuapp.com/api/items?limit=10"
+    url: "${user.K8sBackendUrl}/api/items?limit=10"
```

Comment:
we don't need the `https://` here?

Reply:
The full url with the `https` prefix is returned from `user.K8sBackendUrl`